python/labbox_ephys/sorters/mountainsort4.py

```diff
@@ -0,0 +1,59 @@
+import hither as hi
```

Comment:
Wrapper to mountainsort4 algorithm. Note that this takes as input a recording object and gives a sorting object (rather than recording extractor -> sorting extractor). This is important so that the function can be registered as a hither function (operates on serialized data objects)